packages/api/src/__tests__/useP2PAdvertList.spec.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import useInfiniteQuery from '../useInfiniteQuery';
+import { renderHook } from '@testing-library/react-hooks';
+import useP2PAdvertList from '../hooks/p2p/useP2PAdvertList';
+import APIProvider from '../APIProvider';
+
+jest.mock('../useInfiniteQuery');
+
+const mockUseInfiniteQuery = useInfiniteQuery as jest.MockedFunction<typeof useInfiniteQuery<'p2p_advert_list'>>;
+
+describe('useP2PAdvertList', () => {
+    it('should return undefined if there is no response', () => {
```

Comment:
some hooks use `it` and others use `test`. do we have any convention for this?

Reply:
not that I'm aware of, most tests I wrote I used `it` instead of `test`, but as far as I know there's not much difference between them 😅

Reply:
We should standardize one, since the leads started the tests with `test`, i suggest we go with that

Reply:
Yes, [this](https://stackoverflow.com/questions/45778192/what-is-the-difference-between-it-and-test-in-jest) doc says pretty much the same as well. I guess we should just agree on one?